src/objective-c/examples/RemoteTestClient/RemoteTest.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.9'
 
   # Run protoc with the Objective-C and gRPC plugins to generate protocol messages and gRPC clients.
-  s.dependency "!ProtoCompiler-gRPCPlugin", "~> 1.0.0-pre1.2"
+  s.dependency "!ProtoCompiler-gRPCPlugin"
```

Comment:
As this and `../../tests/RemoteTestClient/RemoteTest.podspec` are used only for testing, the `Podfile` always says to fetch the podspec from the local directory (whatever version that is). So specifying a version here is only giving us more work.